CI is reporting   
```
-- avg_time = 1.7336015939712524
  -- expected_avg_time = 1.4 with err_tol = 0.3
>               assert ok, " !! ERROR !! Regression detected"
E               AssertionError:  !! ERROR !! Regression detected
E               assert False
```
But I think this is actually unrelated as I had already put up a PR to update that manual value.
Let's use this PR to fix it https://github.com/vllm-project/vllm/pull/17064. 

@alexm-redhat I think it's been some time since the last update right?

--- a/vllm/v1/sample/tpu/sampler.py
+++ b/vllm/v1/sample/tpu/sampler.py
@@ -28,7 +28,8 @@ class Sampler(nn.Module):
         # temperature scaling) for the top-k logprobs.
         # This is different from the V0 sampler, which uses the logits that
         # is used for sampling (after penalties and temperature scaling).
-        raw_logprobs = self.compute_logprobs(logits)
+        if sampling_metadata.logprobs:
+            raw_logprobs = self.compute_logprobs(logits)
 
         # Use float32 for the logits.
         logits = logits.to(torch.float32)

--- a/vllm/v1/worker/tpu_model_runner.py
+++ b/vllm/v1/worker/tpu_model_runner.py
@@ -789,7 +789,8 @@ class TPUModelRunner:
             logits = self.structured_decode(require_struct_decoding,
                                             grammar_bitmask_padded, logits,
                                             arange)
-        selected_token_ids, logprobs = self.sample_from_logits(logits, tpu_sampling_metadata)
+        selected_token_ids, logprobs = self.sample_from_logits(
+            logits, tpu_sampling_metadata)
         # Remove padding on cpu and keep dynamic op outside of xla graph.
         selected_token_ids = selected_token_ids.cpu()[:num_reqs]
         logprobs_lists = logprobs.tolists() \
@@ -1256,7 +1257,8 @@ class TPUModelRunner:
     @torch.compile(backend="openxla", fullgraph=True, dynamic=False)
     def sample_from_logits(
             self, logits: torch.Tensor,
-            sampling_metadata: TPUSupportedSamplingMetadata) -> torch.Tensor:
+            sampling_metadata: TPUSupportedSamplingMetadata) -> \
+                tuple[torch.Tensor, Optional[LogprobsTensors]]:
         """
         Sample with xla-friendly function. This function is to be traced 
         separately from `forward` for lighter compilation overhead.

--- a/vllm/v1/worker/tpu_model_runner.py
+++ b/vllm/v1/worker/tpu_model_runner.py
@@ -1256,6 +1256,10 @@ class TPUModelRunner:
     def sample_from_logits(
             self, logits: torch.Tensor,
             sampling_metadata: TPUSupportedSamplingMetadata) -> torch.Tensor:
+        """
+        Sample with xla-friendly function. This function is to be traced 
+        separately from `forward` for lighter compilation overhead.
+        """
         if sampling_metadata.all_greedy:
             out_tokens = torch.argmax(logits, dim=-1, keepdim=True)
             # TODO skip if not needed and compile for it

